.github/styles/config/vocabularies/Base/accept.txt

```diff
@@ -180,3 +180,4 @@ yml
 [xX]DS|xds
 Zipkin
 zsh
+MeshExternalService
```

Comment:
this list is alphabetical, please put it in the right place

Reply:
fixed